lib/App/Config/Chronicle.pm

```diff
@@ -393,6 +409,396 @@ sub _add_dynamic_setting_info {
     return;
 }
 
+=head1 SUBROUTINES/METHODS
+######################################################
+###### Start new API
+######################################################
+
+=head2 local_caching
+
+If local_caching is set to the true then key-value pairs stored in Redis will be cached locally.
+
+Calling update_cache will update the local cache with any changes from Redis.
+refresh_interval defines (in seconds) the minimum time between seqequent updates.
+
+Calls to get on this object will only ever access the cache.
+Calls to set on this object will immediately update the values in the local cache and Redis.
+
+=cut
+
+has local_caching => (
+    isa     => 'Bool',
+    is      => 'ro',
+    default => 0,
+);
+
+=head2 update_cache
+
+Loads latest values from data chronicle into local cache.
+Calls to this method are rate-limited by <refresh_interval>.
```

Comment:
This `<...>` markup probably wanted a formatting character to control it; perhaps `C<refresh_interval>` for fixed-width ("code"), or `I<..>` for italics